modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -48,7 +48,8 @@ def submit
           # This kicks off the first of three jobs required to fully establish the claim
           process_claim(auto_claim)
 
-          render json: auto_claim, status: :accepted, location: "#{request.url[0..-4]}claims/#{auto_claim.id}"
+          render data: { json: form_attributes }, status: :accepted,
+                 location: "#{request.url[0..-4]}claims/#{auto_claim.id}"
```

Comment:
I'm not seeing any response with this line this way.   I think the render of `json:` for the type of data returned needs to be there.  With the line formatted like this `render json: { data: form_attributes }, ...`  I do see the form_attributes returned wrapped in a data object like below.

```
{
    "data": {
        "claimantCertification": true,
        "claimProcessType": "STANDARD_CLAIM_PROCESS",
        "claimDate": "2020-12-18",
...
```